packages/hawtio/package.json

```diff
@@ -46,7 +46,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-markdown": "^8.0.5",
-    "react-router-dom": "^5.3.4",
+    "react-router-dom": "^6.8.0",
```

Comment:
This causing fundamental test failures. Will need to do manual upgrade of `react-router-dom` and resubmit PR.

Reply:
Yes, I filed https://github.com/hawtio/hawtio-next/issues/62 to track the task. Currently @mmelko is working on it.